### DISCUSSION TITLE
Petition against mythic dungeons - Priority for raids and world bosses

### DISCUSSION THREAD

**Discussion by georgeclamnis** [chromiecraft/chromiecraft#5863] [Polls]
Dear community members,

We, the undersigned, hereby express our disapproval and refusal of mythic dungeons. We firmly believe that raids and world bosses should be our main priority when it comes to game content.

While mythic dungeons may have generated excitement among some players, we believe they do not meet the expectations and interests of the majority of our community. They can be seen as a distraction that diverts attention from more significant and thrilling group activities such as raids and encounters with world bosses.

We demand a revision of the current direction of the game in favor of strengthening raids and the availability of world bosses. These elements provide collective challenges and epic experiences that bring players together and create true camaraderie within our community.

Therefore, we urge the game development team to take our concerns into consideration and act accordingly by giving increased attention to raids and world bosses. We are confident that this decision will significantly enhance the game's quality and strengthen our commitment to it.

By signing this petition, we hope to make our collective voice heard and positively influence the future of the game. We call upon all players who share our viewpoint to join us in signing this petition.

Together, we can ensure that our preferences are taken into account and that the game content truly reflects our desires as a community.

Please sign below to show your support for this petition.

**Comment by elthehablo** [comment]
With all due respect, this is written assuming that the development of mythic content stands in the way of scripting raids and world. It hardly is. The development of the module has been developed alongside fixing bugs in current content in part by devs that are not that active with bug fixing current content. This means that hardly any time was lost getting new content quicker because the devs that were working on this were not going to work on dungeons and raid content anyway.

Could make this a very long story and explain it all in depth to you but I'd need a whiteboard and markers.

A strong no from me, and I see the likelihood of this being taken into consideration as very slim.

**Reply by georgeclamnis** [reply]
I understand that you believe the development of mythic content did not interfere with the creation of raids and world content. However, I firmly believe that making decisions without seeking the community's input is a mistake. While the server creates content while considering the community's opinion, I don't think that has been the case in this situation. Mythic dungeons were never a community request in CC (unless I'm mistaken), and everyone was eagerly anticipating raids. You can see this on the Discord channel, where people constantly ask when the raids will be released. I believe it is a bad idea to launch mythic dungeons and claim that we are following Blizzard's lead. Blizzard made several mistakes by not listening to the community, resulting in the loss of many players, investment, and other consequences that you are aware of without me listing them.

**Reply by Zelfie** [reply]
Even if it didn't take any dev time whatsoever, did anyone ask for this?

**Reply by Amrx2** [reply]
Last reply on this for me, yes. It has been on the PTR and pushed for a while. By far and away the largest request was selectable difficulty on raids by players during vanilla, let the casuals / PUGs have raids as they were back in the day, let guilds and 'hardcore' raiders go for the difficult clears. Hence why we announced selectable difficulty all the way back in October 2022 for TBC.

The module needs extensive testing before being deployed for raids, hence why mythic dungeons became a thing so we can test out the functionality on live before raids (also why heroics are getting a detune to be more pug friendly).

**Reply by elthehablo** [reply]
> I understand that you believe the development of mythic content did not interfere with the creation of raids and world content. However, I firmly believe that making decisions without seeking the community's input is a mistake. While the server creates content while considering the community's opinion, I don't think that has been the case in this situation. Mythic dungeons were never a community request in CC (unless I'm mistaken), and everyone was eagerly anticipating raids. You can see this on the Discord channel, where people constantly ask when the raids will be released. I believe it is a bad idea to launch mythic dungeons and claim that we are following Blizzard's lead. Blizzard made several mistakes by not listening to the community, resulting in the loss of many players, investment, and other consequences that you are aware of without me listing them.

You are making parallels to a billion dollar corporation where decisions are largely made by strongly considering what the shareholders find interesting/what can make the company more money to the dev team of a small world of warcraft server. In my opinion that comparison is not really fair. In my experience the staff of CC do listen to the player base. For instance, when instanced world bosses eventually became a thing because one guild was hoarding them all for themselves, which is quite doable on a small to mid population server. And as stated by construct, the mythic mode (then named hardmode) was already an announced feature since the release of TBC. I understand you may be new to this server and missed it, but I also do not think it has not been advertised enough. As stated above it has been live on the PTR for a while and plenty of time has been had for people to criticise it. Don't get me wrong, you are ofc free to criticise it now, and looking at this post you are apparently not alone. But it comes at a bit of an akward spot with it already coming next week. I just wonder why this hasn't been made earlier.

Anyway, to finish this all up, I am not affiliated with the staff nor do I get paid to write this. I am a player in a decently sized raiding guild and I do welcome this feature myself.

**Comment by Amrx2** [comment]
> Dear community members,
> 
> We, the undersigned, hereby express our disapproval and refusal of mythic dungeons. We firmly believe that raids and world bosses should be our main priority when it comes to game content.
> 
> While mythic dungeons may have generated excitement among some players, we believe they do not meet the expectations and interests of the majority of our community. They can be seen as a distraction that diverts attention from more significant and thrilling group activities such as raids and encounters with world bosses.
> 
> We demand a revision of the current direction of the game in favor of strengthening raids and the availability of world bosses. These elements provide collective challenges and epic experiences that bring players together and create true camaraderie within our community.
> 
> Therefore, we urge the game development team to take our concerns into consideration and act accordingly by giving increased attention to raids and world bosses. We are confident that this decision will significantly enhance the game's quality and strengthen our commitment to it.
> 
> By signing this petition, we hope to make our collective voice heard and positively influence the future of the game. We call upon all players who share our viewpoint to join us in signing this petition.
> 
> Together, we can ensure that our preferences are taken into account and that the game content truly reflects our desires as a community.
> 
> Please sign below to show your support for this petition.

One of our contributors in ElDiablo/ elthehablo has summarised it up perfectly below, nothing further to add really other than confirming what they said. This is in place to ensure we can meet the commitment we made in October last year that TBC raids will have two versions in both 'normal' (light tuned, pug & casual friendly like ZG & AQ20 in Vanilla) and what is now called 'Mythic' mode (what was just announced by Honey) to appeal to all parts of our community.

**Comment by Zelfie** [comment]
Every day we stray further from QOL Blizzlike and more to mickey mouse fun server.
Instant 255 when?
Even if it didn't take any dev time, it's a slippery slope that has divided the population and has players looking at other projects. Population is already pretty low.

**Comment by elthehablo** [comment]
...lest we forget the positive points mythic brings. Allowing for more difficult content so well organised guilds don't blast through everything as fast whilst still keeping the option available for more casual players. Having raided here all through vanilla, I can say that a change like this is very welcome.

**Comment by rheshs** [comment]
i would suggest you guys from the dev team to stop advertising the server as a ''blizzlike server'' cause this is not the case

**Reply by Amrx2** [reply]
That was dropped somewhere during raids in Vanilla well over a year ago?

**Reply by rheshs** [reply]
buffed content is nothing like adding costum gear/new boss mechanic

**Reply by elthehablo** [reply]
> i would suggest you guys from the dev team to stop advertising the server as a ''blizzlike server'' cause this is not the case

AC is a blizlike 3.3.5a project. CC is a progressive server using AC as a base as a project to help fix bugs in vanilla/TBC content in the AC core

**Reply by Zelfie** [reply]
_The word Blizzlike was on the Discord welcome page up until a few hours ago :')_
EDIT
I am wrong. lol

**Reply by Amrx2** [reply]
Completely optional content I would add. The same way our challenge modes are optional, as is the Transmog module.

**Reply by Amrx2** [reply]
An oversight as it was removed everywhere else before, feel free to check out the webpage using the Way back machine otherwise.

**Reply by Synful-Syn** [reply]
The word Blizzlike was removed nearly a year ago. Stating it was removed a few hours ago is wrong. Screenshot with east coast time.
![image](https://github.com/chromiecraft/chromiecraft/assets/22718174/a07de205-9687-46eb-8d11-b02bf2fab46b)

**Reply by Zelfie** [reply]
I'll own that. I must've taken a screenshot post I saw in Discord out of context and not done my appropriate research properly.

**Reply by Zelfie** [reply]
However you must respect that everyone that rolled on this server when it was a QOL Blizzlike, will have their own thoughts about it getting further and further away from that.
If I knew this is where we'd be, I wouldn't have.

**Comment by elthehablo** [comment]
> Every day we stray further from QOL Blizzlike and more to mickey mouse fun server.
> Instant 255 when?
> Even if it didn't take any dev time, it's a slippery slope that has divided the population and has players looking at other projects. Population is already pretty low.

Slippery slope is a logical fallacy. Wotlk client TBC simply doesn't have the same feel as classic TBC has, and therefore has some tuning to make it more entertaining for players. A slower than usual progression makes some players bored and the QOL changes and some extra content does help with that. For me, at least.

**Comment by InvictaChromie** [comment]
doesn't feel very blizzlike tho which I thought was part of the point of the server? 

also hoping the mythic bonus enchanted stuff won't be very impactful,
they should only be for a fun challenge I feel anyway !

**Reply by elthehablo** [reply]
They add the equivalent of an uncommon gem slotted AFAIK. Also see above thread for blizzlike discussion

**Reply by elthehablo** [reply]
Is it gonna help dpswise? Yes. Can it skew tuning a slight bit? Yes. Is tuning with the 80 talents at 70 perfect anyway so does this matter much? No.

**Reply by Amrx2** [reply]
That's the aim, overall gear benefit is minimal, at about a 1.5-2% increase with fixed bonuses at a cost of much harder clears. There's +7SP, +14AP or +8 dodge, depending on the kind of the item.

Again this is entirely optional and is aimed at those who loved the challenges we provided with the harder vanilla raids we did (such as AQ40)

**Reply by Never6** [reply]
7sp isn't enough to replace some items

**Reply by Wosser** [reply]
I'm strongly in the blizzlike-camp, but I don't mind these mythic dungeons and I don't mind an RNG-free way to get the heroic loot. What I really dislike is the "enchanted" custom loot. With the item design in TBC, some classes/speccs have a hard time finding meaningful upgrades in T5 content, adding one uncommon gem to HC dungeon gear might be enough to make T4 content rather irrelevant for these speccs too, meaning if you farm all the best mythic gear your character won't get stronger til Mount Hyjal, which might be a year away. So I'd like the devs to take away these special enchanted items and instead let us buy rare HC loot too, many BiS items for many classes aren't epic atm.

**Comment by Amrx2** [comment]
Whilst we appreciate the suggestion, this isn't a suggestion we will consider. Please see above responses from myself & @elthehablo regarding Dev time usage / reasonings for this. Cheers.

**Comment by InvictaChromie** [comment]
> That's the aim, overall gear benefit is minimal, at about a 1.5-2% increase with fixed bonuses at a cost of much harder clears. There's +7SP, +14AP or +8 dodge, depending on the kind of the item.
> 
> Again this is entirely optional and is aimed at those who loved the challenges we provided with the harder vanilla raids we did (such as AQ40)

we'll see I guess, can see a lot of players feeling forced to do it for "bis reasons" even if it is max 2% difference like you say lol

**Reply by elthehablo** [reply]
If you are forcing yourself to get this gear you are doing it wrong. My point is that nobody is forcing people to gear their chars and if you are a minmaxer and you like hard content you will be going for this anyway.

**Reply by KostovDimitar** [reply]
I am a min maxer and i dont like custom gear. Will i do this? No. Will i look for a new server? Yes.

**Comment by Never6** [comment]
Closed already?

Imo mythic tuned content is fine. 
Seems a little unwarranted because why not just make heroics as hard as you want mythics to be 

The issue is rewards
I think a far better option is, instead of 1 enchant per item it's a gem that players can choose
7sp isn't much on one item, sure it's a lot if you replace every piece but that isn't guaranteed. Not to mention some classes don't even use all epic pieces from heroics if any even do.
People being able to pick gem is better for a couple reasons
1. Players can choose what they want.
It's unfair for say a warrior to get 14ap when 8str or 13 crit rating gems are better.
8str is prob 20ap in full buffs
8dodge rating is also bad, from a bear main perspective. Dodge rating is inferior to agi per point not to mention you get more than just % dodge  from agility.
Even if bears go full avoidance and get their Stam from gear not gems agility is better without question. And it doesn't take a bear main of 10 years to tell you that.
Having +1 gem means blue gear get buffed which most classes use at least some dungeon pieces ( at least the classes I play do) and since you can pick gem you might be able to change gear
Say a plate tank replaces 2 pieces of heroic gear with mythic gear and they get +2 gem slots, they use defense rating gems. This is what? +20defense for free? Now they might be 20points over cap which can let them potentially change gear slots for more threat or more stam
Off topic but
This content seems like something players should know about sooner than 3 days before implementation 
Cause say majority hate it, what do you do?
Scrap the idea? You just wasted your time then.
If players knew about it sooner, you could get a feeling for how players feel about it and figure out if it's worth investi ng time into.
Cause now you box yourself into a corner. It's too late to change anything no amount of critic or backlash is going to change anything

**Comment by Amrx2** [comment]
For those who want to discuss the loot given in Mythics - https://github.com/chromiecraft/chromiecraft/discussions/5865
